.github/workflows/pytest.yml

```
@@ -35,6 +42,14 @@ jobs:
           python -m pip install --upgrade pip -r requirements-dev.txt
           pip install -e .
 
+      - name: Downgrade git to the Ubuntu official repository's version
+        if: ${{ matrix.runner == 'ubuntu-20.04' && matrix.python-version == '3.8' }}
```

```suggestion
        if: ${{ matrix.runner == 'ubuntu-20.04' && matrix.python-version == '3.8' }}
        env:
          GITHUB_TOKEN: ${{ github.token }}
```

Is this the scope it's needed in?